modules/webrtc/doc_classes/WebRTCMultiplayer.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="WebRTCMultiplayer" inherits="NetworkedMultiplayerPeer" category="Core" version="3.2">
+	<brief_description>
+		A simple interface to create a peer-to-peer mesh network composed of [WebRTCPeerConnection] that is compatible with the [MultiplayerAPI].
+	</brief_description>
+	<description>
+		This class constructs a full mesh of [WebRTCPeerConnection] (one connection for each peer) that can be used as a [member MultiplayerAPI.network_peer].
+		You can add each [WebRTCPeerConnection] via [method add_peer] or remove them via [method remove_peer]. Peers must be added in [constant WebRTCPeerConnection.STATE_NEW] state to allow it to create the appropriate channels. This class will not create offers nor set descriptions, it will only poll them, and notify connections and disconnections.
+		[signal NetworkedMultiplayerPeer.connection_succeeded] and [signal NetworkedMultiplayerPeer.server_disconnected] will not be emitted unless [code]server_compatibility[/code] is [code]true[/code] in [method initialize]. Beside that data transfer works like in a [NetworkedMultiplayerPeer].
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="add_peer">
+			<return type="int" enum="Error">
+			</return>
+			<argument index="0" name="peer" type="WebRTCPeerConnection">
+			</argument>
+			<argument index="1" name="peer_id" type="int">
+			</argument>
+			<argument index="2" name="unreliable_lifetime" type="int" default="1">
+			</argument>
+			<description>
+				Add a new peer to the mesh with the given [code]peer_id[/code]. The [WebRTCPeerConnection] must be in state [constant WebRTCPeerConnection.STATE_NEW].
+				Three channels will be created for reliable, unreliable, and ordered transport. The value of [code]unreliable_lifetime[/code] will be passed to the [code]"maxPacketLifetime"[/code] option when creating unreliable and ordered channels (see [method WebRTCPeerConnection.create_data_channel]).
```

Comment:
No need to double-quote maxPacketLifeTime here, it is already surrounded with [code][/code] :)